ledger/notary-plugin/src/main/kotlin/net/corda/v5/ledger/notary/pluggable/NotarisationPayload.kt

```diff
@@ -0,0 +1,15 @@
+package net.corda.v5.ledger.notary.pluggable
+
+/**
+ * Container for the transaction and notarisation request signature. This is the payload that gets sent by a client to
+ * a notary service for committing the input states of the [transaction].
+ *
+ * This interface uses Any as for the [transaction] type. The [transaction] object will be validated as a concrete type
+ * (for example [net.corda.v5.ledger.utxo.transaction.UtxoSignedTransaction]) when a class inherits from this interface
+ * with a specified [validTypes] list.
```

Comment:
```suggestion
 * This interface uses Any for the [transaction] type. The [transaction] object will be validated as a concrete type
 * (for example [UtxoSignedTransaction][net.corda.v5.ledger.utxo.transaction.UtxoSignedTransaction]) against the specified [validTypes] list.
 *
```

- I feel this is clearer that we are validating the transaction against the type, rather than the inheriting class.
- Doc will display `UtxoSignedTransaction` (neater) but still have a valid link using the fully qualified name